### PR TITLE
Networks

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity Good to merge?

File: src/containers/network/NetworkCommunities.js [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+
+@prefetch(({ dispatch, params: { id }, query }) => dispatch(fetch(subject, id, query)))
+@connect((state, { params: { id }, location: { query } }) => {
+  return {
+    ...connectedListProps(state, {subject, id, query}, 'communities'),
+    network: state.networks[id],
+    currentUser: state.people.current
+  }
+})
+export default class NetworkMembers extends React.Component {
```
Comment (levity) [inline src/containers/network/NetworkCommunities.js]: s/Members/Communities/

Comment (levity) [inline src/containers/network/NetworkCommunities.js]: this can be stateless

**Comment by levity** [comment]
looks good otherwise

**Pull request merged**